lib/Controller/SettingsController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Survey Server
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the LICENSE.md file.
+ *
+ * @author Marcel Scherello <surveyserver@scherello.de>
+ * @copyright 2023 Marcel Scherello
+ */
+
+namespace OCA\Survey_Server\Controller;
+
+use OCA\Survey_Server\Service\SettingsService;
+use OCP\AppFramework\Controller;
+use OCP\AppFramework\Http\DataResponse;
+use OCP\IRequest;
+use Psr\Log\LoggerInterface;
+
+class SettingsController extends Controller
+{
```

Comment:
```suggestion
class SettingsController extends Controller {
```

Our coding style is different, but we can fix this later on, by adding https://github.com/nextcloud/coding-standard